eventuals/task.h

```diff
@@ -166,18 +166,32 @@ struct _TaskFromToWith {
     Fail = 2,
   };
 
-  template <typename K_, typename From_, typename To_, typename... Args_>
+  template <
+      typename K_,
+      typename From_,
+      typename To_,
+      typename... Args_>
   struct Continuation {
```

Comment:
Since this has non-trivial methods, the [Google C++ style guide](https://google.github.io/styleguide/cppguide.html#Structs_vs._Classes) would say that this should be a `class` rather than struct.

Probably beyond the scope of this review, though, since it might apply to many files.

Reply:
Will refactor in another commit